content/articles/getting-started-with-kali-linux/index.md

```diff
@@ -0,0 +1,290 @@
+### Getting started with Kali Linux
+
+**Kali Linux** is the most popular penetration testing platform that provides a variety of security auditing tools used by security specialists and hackers in day-to-day encounters.
+
+Tools available on Kali Linux enable the user to gather information, perform exploits and prevent their devices from exploits.
+
+### Installing Kali Linux
+
+For beginners, it is advised to have your Kali Linux in a **virtual box**. 
+You can have your Kali Linux in your favorite virtual box. Available virtual boxes include:
+
+  * Vmware virtual box
+  * Hyper V virtual box
+  * Kernel-based virtual box 
+  
+For our tutorial, we will use VMware virtual box (My favorite).
+
+#### Download Kali Linux iso
+
+First, before installing the VMware you have to download the Kali Linux iso. Use the link below:
+
+  * Kali Linux iso.
+
```

Comment:
Where is the link? I assume the link was to take the readers to where they can download it. Please add it